external/buildscripts/manifest.stevedore

```diff
@@ -10,7 +10,7 @@ unity-internal: android-ndk-win/r19-unity_799f451638695b9da797fcd509f9f2a8e59e35
 # macOS
 unity-internal: android-ndk-mac/r19-unity_8b169ff2a8234c85e0c5ba3c776aa94273cd3c15fdc96d213154970d87938589.7z
 unity-internal: mac-toolchain-11_0/12.2-12B5018i_351c773fb8a192039fe0f0e962314b888102b0718c734ad54f3906c0caeed1c9.zip
-unity-internal: mono-build-tools-extra/f2f8c2c6e2674cdcac8950643c93915e2631d92e_42ec904e3fc5b604db5fc7430f4a1a3c018a4261e0362dbf3229b7a48589b691.7z
+testing: mono-build-tools-extra/755290e1f53e88bc3a8caa0224993ee91a1e1a02_c9e1fb5ebedba9bf3ccdfd938ae7db739ae8fef93fcb9647a4542bc3749038b5.7z
```

Comment:
We probably don't want to merge this until the mono-build-tools-extra .7z is promoted to unity-internal.

I'd recommend we first confirm that this works end-to-end with the IL2CPP excessive tests (maybe you have already done that @bholmes). Then we can ask for this to be promoted. Once it is promoted, we can reference it from unity-internal.

Reply:
yep, going to test the mono artifacts from the test build with il2cpp.  Once confirmed we need to promote and update.

Reply:
Done